app/models/user.rb

```diff
@@ -17,25 +20,22 @@ class User < ActiveRecord::Base
 
   def self.authenticate(email, password)
     user = find_by_email(email)
-    if user && user.authenticate(password)
-      user
-    else
-      nil
-    end
+    return user if user && user.authenticate(password)
   end
 
   def regenerate_auth_token
     self.auth_token = generate_auth_token
-    return self.auth_token
+    auth_token
   end
 
   private
-    def set_auth_token
-      return if self.auth_token.present?
-      self.auth_token = generate_auth_token
-    end
-
-    def generate_auth_token
-      SecureRandom.uuid.gsub(/\-/,'')
-    end
+
+  def set_auth_token
+    return if auth_token.present?
+    self.auth_token = generate_auth_token
+  end
+
+  def generate_auth_token
+    SecureRandom.uuid.gsub(/\-/, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.